sympy/simplify/tests/test_simplify.py

```diff
@@ -914,3 +914,16 @@ def test_issue_19484():
 def test_issue_19161():
     polynomial = Poly('x**2').simplify()
     assert (polynomial-x**2).simplify() == 0
+
+def test_issue_11004():
+    N=Symbol('N')
+    half = Float('0.5', 4)
+    def f(n):
+        return sqrt(2*pi*n)*(n/E)**n
+    def m(n, k):
+        return  f(n)/ (f(n/k)**k)
+    def p(n,k):
+        return m(n, k)/(k**n)
+    q = (p(n, 2)/p(n, 3)).simplify()
+    z = log(p(n, k)/p(n, k+1)).expand(force=True)
```

Comment:
We don't need all of this to create the expression just get the repr and use that:
```
In [2]: repr(z)
Out[2]: 'k*n*log(n)/(k + 1) - k*n*log(k + 1)/(k + 1) - k*n/(k + 1) + k*log(k)/2 - k*log(k + 1)/2 - n*log(n) + n*log(k + 1) + n + n*log(n)/(k + 1) - n*log(k + 1)/(k + 1) - n/(k + 1) + log(n)/2 - log(k + 1)/2 + log(2)/2 + log(pi)/2'
```
This makes the test more efficient and also more specific in case the output of e.g. simplify/expand changes in future.

Reply:
Thank you for your suggestion, I made the necessary changes.
Kindly suggest if further betterment can be done.
Thanks.